Error with EGS2017 & enabling Radiative Compton corrections

After installing a pristine close of recent EGS2017 master branch I tried to enable Radiative Compton corrections on several user codes.

E.g. with dosxyznrc, prior to adding required line to Makefile I run 'make clean' and 'make' and everything went fine. Then I added the line and run 'make' and got:
Mortran compilation for dosxyznrc_170216.F
mortran3.exe: using following I/O units
  raw/hex data file: /home/ojalajar/EGS2017m/HEN_HOUSE/bin/170216/mortran3.dat
  command line files:
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/src/egsnrc.macros
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/utils/timing.macros
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/lib/170216/machine.macros
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/src/ranmar.macros
  -> appending dosxyznrc_user_macros.mortran
  -> appending srcxyznrc.macros
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/src/transportp.macros
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/src/pegs4_macros.mortran
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/utils/phsp_macros.mortran
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/utils/iaea_phsp_macros.mortran
  -> appending dosxyznrc.mortran
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/src/rad_compton1.mortran
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/src/get_inputs.mortran
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/src/get_media_inputs.mortran
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/lib/170216/machine.mortran
  -> appending srcxyznrc.mortran
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/src/ranmar.mortran
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/utils/nrcaux.mortran
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/src/egs_utilities.mortran
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/src/egs_parallel.mortran
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/src/pegs4_routines.mortran
  -> appending /home/ojalajar/EGS2017m/HEN_HOUSE/src/egsnrc.mortran
  generated mortjob.mortran
  fortran output: dosxyznrc_170216.F
  mortlst output: dosxyznrc_170216.mortlst

  Mortran compiling ...  OK


Fortran compiling dosxyznrc_170216.F using flags ' -fPIC -O2' and extra objects/libs '/home/ojalajar/EGS2017m/HEN_HOUSE/lib/170216/egs_c_utils.o /home/ojalajar/EGS2017m/HEN_HOUSE/lib/170216/read_write_pardose.o /home/ojalajar/EGS2017m/HEN_HOUSE/lib/170216/load_beamlib.o  /home/ojalajar/EGS2017m/HEN_HOUSE/lib/170216/load_vculib.o  -ldl -L/home/ojalajar/EGS2017m/HEN_HOUSE/egs++/dso/170216 -Wl,-rpath,/home/ojalajar/EGS2017m/HEN_HOUSE/egs++/dso/170216 -liaea_phsp'
dosxyznrc_170216.F:4780.22:

      integer*4 medium,lgle,icheck
                      1
Error: Symbol 'medium' at (1) already has basic type of INTEGER
dosxyznrc_170216.F:4866.24:

                  icheck = acheck
                        1
Error: Symbol 'icheck' at (1) has no IMPLICIT type
dosxyznrc_170216.F:4860.18:

              lgle = i
                  1
Error: Symbol 'lgle' at (1) has no IMPLICIT type
make: *** [/home/ojalajar/BEAM2017/bin/170216/dosxyznrc] Error 1

--- a/HEN_HOUSE/src/rad_compton1.mortran
+++ b/HEN_HOUSE/src/rad_compton1.mortran
@@ -163,7 +163,7 @@ character     radc_file*256;
 integer*4     lnblnk1,egs_get_unit;
 $INTEGER      radc_unit, want_radc_unit;
 $INTEGER      nskip,i,j,ne,nu,ny1,ny2,nw,jh,jl;
-$INTEGER      medium,lgle,icheck;
+$INTEGER      lgle,icheck;
 real*8        aux1,gmfp,gmfp_old,gbr1,gbr1_old,gbr2,gbr2_old,gle,frad,
               acheck,acheck1,sum,aux;
 $INTEGER      nx,nbox,ixtot,ibtot;

